liblangutil/Token.h

```diff
@@ -196,6 +196,7 @@ namespace solidity::langutil
 	\
 	/* Ether subdenominations */                                       \
 	K(SubWei, "wei", 0)                                                \
+	K(SubGwei, "gwei", 0)                                              \
```

Comment:
I think this is a breaking change. `gwei` should be parsed as a special case in https://github.com/ethereum/solidity/blob/develop/libsolidity/parsing/Parser.cpp#L1827

Reply:
I still don't understand why

Reply:
Both points. Why it needs to be a special case and why it would be breaking

Reply:
Before the change this is valid:
```
contract C {
  uint constant gwei = 1;
}
```

Reply:
This should also be added as a test case.

Reply:
hmm I see. Will change.

Reply:
How else would you do it if you cannot add a new token?

Reply:
Why can I not add a new token? Because we want `uint constant gwei` to keep working?

Reply:
Because adding a new token is a breaking change, there may be many other kinds of places it will break, the `uint constant gwei` is an example I know of and is potentially used by contracts.

Reply:
Well, adding a new token would be the "proper" way. I guess we should decide whether we want this to be breaking or not. We could also do a non-breaking change first and then make it breaking later, but doing this without adding a token is at the very least hacky. I mean, we want to do more breaking releases, so I think it's fine to just retarget it to `breaking`..

Reply:
I don't think it is so hacky adding an extra case in the parser...

Reply:
I don't think so either.

Reply:
Fair enough, maybe I just expected more parts to be affected. But it seems I can just construct the `Literal` using an `identifier` `"gwei"` .